lib/active_support/cache/redis_store.rb

```diff
@@ -82,7 +82,7 @@ def write(name, value, options = nil)
           if options[:expires_in].present? && options[:race_condition_ttl].present? && options[:raw].blank?
             options[:expires_in] = options[:expires_in].to_f + options[:race_condition_ttl].to_f
           end
-          entry = options[:raw].present? ? value : Entry.new(value, options)
+          entry = options[:raw].present? ? value : Entry.new(value, options.to_h.symbolize_keys)
```

Comment:
if these are actually keyword arguments, I think you'll need to do `**options.to_h.symbolize_keys`.